t/t2000-imp-exec.t

```diff
@@ -0,0 +1,119 @@
+#!/bin/sh
+#
+
+test_description='IMP exec basic functionality test
+
+Basic flux-imp exec functionality and corner case handing tests
+'
+
+# Append --logfile option if FLUX_TESTS_LOGFILE is set in environment:
+test -n "$FLUX_TESTS_LOGFILE" && set -- "$@" --logfile
+. `dirname $0`/sharness.sh
+
+flux_imp=${SHARNESS_BUILD_DIRECTORY}/src/imp/flux-imp
+sign=${SHARNESS_BUILD_DIRECTORY}/t/src/sign
+
+echo "# Using ${flux_imp}"
+
+fake_imp_input() {
+	printf '{"J":"%s"}' $(echo $1 | $sign)
+}
+
+test_expect_success 'flux-imp exec returns error when run with no args' '
+	test_must_fail $flux_imp exec
+'
+test_expect_success 'flux-imp exec returns error when run with only 1 arg' '
+	test_must_fail $flux_imp exec foo
+'
+test_expect_success 'flux-imp exec returns error with no input' '
+	test_must_fail $flux_imp exec shell arg < /dev/null
+'
+test_expect_success 'flux-imp exec returns error with bad input ' '
+	echo foo | test_must_fail $flux_imp exec shell arg
+'
+test_expect_success 'flux-imp exec returns error with bad JSON input ' '
+	echo "{}" | test_must_fail $flux_imp exec shell arg
+'
+test_expect_success 'flux-imp exec returns error with bad JSON input ' '
```

Comment:
duplicate of previous test?

Reply:
I think I had meant to echo some invalid JSON in the 2nd test, thanks for catching this!